java/google/registry/rde/imports/XjcToContactResourceConverter.java

```diff
@@ -90,6 +98,23 @@ static ContactResource convertContact(XjcRdeContact contact) {
         .build();
   }
 
+  private static HistoryEntry createHistoryEntry(XjcRdeContact contact) {
+    XjcRdeContactElement element = new XjcRdeContactElement(contact);
+    final HistoryEntry historyEntry =
+        new HistoryEntry.Builder()
+            .setType(HistoryEntry.Type.RDE_IMPORT)
+            .setClientId(contact.getClID())
+            .setTrid(Trid.create(null))
```

Comment:
Might as well put in a client trid that identifies this as belonging to the import